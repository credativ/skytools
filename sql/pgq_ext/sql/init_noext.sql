\set ECHO none
\i structure/install.sql

