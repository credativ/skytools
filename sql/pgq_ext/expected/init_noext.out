\set ECHO none
 upgrade_schema 
----------------
              4
(1 row)

